.github/workflows/gh-pages.yaml

```diff
@@ -7,6 +7,8 @@ on:
     paths:
       - '**.md'
       - 'docs/**'
+permissions:
+  contents: read
```

Comment:
i think we need write here, the implicit GITHUB_TOKEN is being used by EndBug/add-and-commit

Reply:
we are setting 
```
    permissions:
      contents: write
```
on a job level